lti_consumer/api.py

```diff
@@ -52,3 +57,35 @@ def get_lti_consumer(config_id=None, block=None):
     # Return an instance of LTI 1.1 or 1.3 consumer, depending
     # on the configuration stored in the model.
     return lti_config.get_lti_consumer()
+
+
+def get_lti_1p3_launch_info(config_id=None, block=None):
+    """
+    Retrieves the Client ID, Keyset URL and other urls used to configure a LTI tool.
+    """
+    if not config_id and not block:
+        raise Exception("You need to pass either a config_id or a block.")
+
+    # Empty reponse
+    launch_info = {}
+
```

Comment:
There's no reason to have these lines: the only return comes after we create `launch_info` as a full dict.